README.md

```diff
@@ -1,4 +1,4 @@
-# CKEditor 4 WYSIWYG editor Angular component
+# CKEditor 4 component for Angular
```

Comment:
```suggestion
# CKEditor 4 WYSIWYG editor Angular component
```

The `CKEditor 4 WYSIWYG editor Angular component` is SEO optimized title so we shouldn't change it ;) I'm reverting this change back.